<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureEmulator</class>
 <widget class="QMainWindow" name="TemperatureEmulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TemperatureEmulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Настройка</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelCOM">
         <property name="text">
          <string>COM-порт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbCOM">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelTEMP">
         <property name="text">
          <string>Температура</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sbTEMP">
         <property name="suffix">
          <string>°C</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-4.000000000000000</double>
         </property>
         <property name="maximum">
          <double>65.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>27.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rndButton">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Случайное значение</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>42</red>
            <green>42</green>
            <blue>42</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>42</red>
            <green>42</green>
            <blue>42</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled/>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
